src/executeQuery.js

```diff
@@ -0,0 +1,88 @@
+'use strict'
+const sql = require('./sql')
+const Query = require('./query')
+const { calculateClassBreaks, calculateUniqueValue } = require('./generateBreaks/breaks')
+const _ = require('lodash')
+
+function breaksQuery (features, query, options) {
+  // TODO: add check if query is valid (or should this have been handled in query?)
+  const queriedData = standardQuery(features, query, options)
+  if (queriedData === undefined || queriedData.features === undefined) throw new Error('query resposne undefined')
+  if (queriedData.features.length === 0) throw new Error('need features in order to classify')
+
+  try {
+    const classification = options.classificationDef
```

Comment:
I'm not a fan of wrapping large chunks of code in try catch, what's your reasoning here?